tmt/steps/prepare/feature/fips-enable.yaml

```diff
@@ -0,0 +1,196 @@
+---
```

Comment:
Can we add a comment here to some official docs, where one could get more information about FIPS?